xbmc/LangInfo.cpp

```diff
@@ -274,6 +272,14 @@ void CLangInfo::CRegion::SetGlobalLocale()
     strLocale += ".UTF-8";
 #endif
   }
+  return strLocale;
+}
+
+// set the locale associated with this region global. This affects string
```

Comment:
Github changes can trick you but that comments was already there..
It's not a problem to move them.. In a separate commit maybe?